src/sidebar/store/modules/test/filters-test.js

```diff
@@ -113,6 +157,100 @@ describe('sidebar/store/modules/filters', () => {
       });
     });
 
+    describe('getFilter', () => {
+      it('returns the filter you ask for', () => {
+        store.setFilter('five', { value: 'cinq', display: '5' });
+
+        const fiveFilter = store.getFilter('five');
+
+        assert.equal(fiveFilter.value, 'cinq');
+        assert.equal(fiveFilter.display, '5');
+      });
+
+      it('returns undefined if nonesuch filter', () => {
```

Comment:
```suggestion
      it('returns undefined if nonesuch filter', () => {
```

Did you mean "no such filter"?